pkg/endpoint/conntrack.go

```diff
@@ -81,7 +81,10 @@ func RunGC(e *endpoint.Endpoint, isLocal, isIPv6 bool, filter *ctmap.GCFilter) {
 }
 
 // EnableConntrackGC enables the connection tracking garbage collection.
-func EnableConntrackGC(ipv4, ipv6 bool) {
+func EnableConntrackGC() {
+	ipv4 := !config.AgentConfig().IPv4Disabled
```

Comment:
I just realized this now. `IPv4Disabled` is a configuration set up when starting the daemon and never changes. How do we, as developers, know the difference of a variable that can change over time, such as `debug`, and a non-changing one like `IPv4Disabled`? I'm assuming the difference is between the use of methods for changing variables (`debug` -> `DebugEnabled()`) vs the direct access of variables for non-changing ones (`IPv4Disabled`)?

Reply:
So far I have been assuming that only `Opts` can be mutated. This is how I documented it in the code as well now.